app/models/visit.rb

```diff
@@ -22,12 +25,14 @@ def validate_amount_of_adults
       errors.add :base, "There must be at least one adult visitor"
     end
     if visitors.count { |v| v.age && v.age >= adult_age } > 3
-      errors.add :base, "You can book a maximum of 3 visitors over the age of #{adult_age} on this visit"
+      errors.add :base, "You can book a maximum of 3 visitors over the age of \
```

Comment:
Again.. you'll end up with strings like:

"You can book a maximum of 3 visitors over the age of         18 on this visit"